.circleci/config.yml

```diff
@@ -940,8 +940,6 @@ workflows:
               ignore:
                 - /pull\/.*/
                 - master
```

Comment:
https://github.com/determined-ai/determined/blob/master/.circleci/config.yml#L1000
https://github.com/determined-ai/determined/blob/master/.circleci/config.yml#L1011

we need to change the requires in the above locations to also require `build-and-push-docker-images`!